src/main/java/bio/terra/service/snapshotbuilder/query/filtervariable/BooleanAndOrFilterVariable.java

```diff
@@ -16,6 +17,15 @@ public String renderSQL() {
         .collect(Collectors.joining(" " + operator.renderSQL() + " ", "(", ")"));
   }
 
+  @Override
+  public List<TableVariable> getTables() {
+    return subFilters.stream()
+        .map(FilterVariable::getTables)
+        .flatMap(List::stream)
+        .distinct()
```

Comment:
If this result is further combined upstream then it will need to be `distinct()`'d again. If that happens, it would be better to only call `distinct()` once at the end.

Alternatively, if the order of the result is unimportant it could be `Set<TableVariable>` instead of `List` and then we wouldn't need to worry about calling `distinct()`.

Reply:
I am actually going to pull the `distinct()` call - I realized its a lot more complicated than that. Because we might join against tables multiple times, how do we tell if it is distinct? I think this ultimately was premature optimization.